styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -170,6 +170,10 @@ void tick() {
     updateStats(resources, currentResourceUsage);
   }
 
+  private boolean isConsumingResources(State state) {
+    return state != State.NEW && state != State.QUEUED;
```

Comment:
Should we also exclude `TERMINATED`, `FAILED`, `ERROR`, and `DONE` ?

Reply:
We could, although as @honnix mentioned once, those are transient states and should quickly get removed. If they don't get cleaned up then that's a reason for concern, and it's good to be able to see it (although maybe with another metric). 
Also the terminal states don't cause spikes in currentResourceUsage the way `State.NEW` does.

Reply:
Yeah, let's keep it simple. Those transient states will go away fairly quickly, and if they don't we might be having some trouble and in that case be conservative on resource usage might be better.

Reply:
SGTM, how about a comment saying that?

Reply:
Will fix.